report.py

```diff
@@ -48,6 +50,35 @@ def get_translations(cls):
         return JinjaTranslator(lambda text: translate(text))
 
 
+class Jinja2ReportLogo(Jinja2Report):
+
+    @classmethod
+    def render(cls, action, context):
+        context['logo'] = cls.get_logo(context['user'])
+        return super().render(action, context)
+
+    @staticmethod
+    def get_logo_filename():
+        return 'logo.png'
```

Comment:
I'd expect this to be stated at the template level, so if somebody crafts a template they don't need to further extend and register python classes.

Reply:
Assuming your thesis that is better not to hardcode as 'logo' any attachment (and I agree), probably this function must be removed. The purpose of this function is just to give a default filename. Anyway I agree that is fine to be expected at the template level, so I'm removing it.